api/v1/helpers/verbs/patchUtils.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * api/v1/helpers/verbs/patchUtils.js
+ */
+'use strict'; // eslint-disable-line strict
+
+const featureToggles = require('feature-toggles');
+const u = require('./utils');
+const publisher = u.publisher;
+const event = u.realtimeEvents;
+const httpStatus = require('../../constants').httpStatus;
+const constants = require('../../../../cache/sampleStore').constants;
+const redisModelSample = require('../../../../cache/models/samples');
+const helper = require('../nouns/perspectives');
+const redisCache = require('../../../../cache/redisCache').client.cache;
```

Comment:
few this required here are not actually used in this file. So can we please remove them?

Reply:
yep removing them now.